wordpress-mobile/WordPress-Android
# Jetpack Section: Backup Screen Tracking Events

Parent #13629

This PR adds five new track events specific to the new `Backup` screen and enhances an existing `Activity Log Detail` event with a `source` property so that it differentiates between the `activity_log` or `backup` entry points.

To test:
- Launch app and go to `My Site` tab.
- Got to: `My Site` -> `Toolbar Avatar` -> `App Settings` -> `Privacy settings`
- Switch on `Collect information` to enable tracking on your device.
- Go to `My Site` -> `Toolbar Avatar` -> `App Settings` -> `Test feature configuration`.
- Enable `BackupsFeatureConfig`, scroll down and click the `RESTART THE APP` button. **DO NOT** confuse this feature flag with the `BackupFeatureConfig` (singular). Make sure to enable the right feature flag.
- Open `Android Logcat`, filter the logcat by `tracked` and start noticing the `🔵 Tracked` events.
- Go back to `My Site` and click on the `Backup` menu item. Notice the `jetpack_backup_list_opened` track event is triggered.
- Find a rewindable item and click on it. Notice the `activity_log_detail_opened` track event is triggered with `backup` as its `source`.
- Click the `RESTORE` button and then the `RESTORE SITE` dialog button. Notice that you will be redirected back to the `Backup` screen and that the `jetpack_backup_rewind_started` track event is triggered.
- Click the `Date Range` chip. Notice the `jetpack_backup_filterbar_range_button_tapped` track event is triggered.
- Select a date range and click `SAVE`. Notice that you will be redirected back to the `Backup` screen and that the `jetpack_backup_filterbar_select_range` track event is triggered.
- Click the `x` inner button within the chip to clear the filter. Notice the `jetpack_backup_filterbar_reset_range` track event is triggered.

PR submission checklist:
